Tests/Unit/MiniAppSecureStorageTests.swift

```diff
@@ -283,7 +283,7 @@ class MiniAppSecureStorageTests: XCTestCase {
             }
         }
 
-        wait(for: [removeExpectation], timeout: 10.0)
+        wait(for: [removeExpectation], timeout: 30.0)
```

Comment:
Any reason to increase the timeout? I think already 10 seems to be a bigger timeout. Also, we don't have any network APIs here.

Reply:
because of the bitrise downgrade